code/obj/item/device/pda2/pda2.dm

```diff
@@ -378,6 +378,9 @@
 		if(ismob(src.loc))
 			var/mob/mob = src.loc
 			get_all_character_setup_ringtones()
+			if(mob.mind)
```

Comment:
Questionable place to put this, might get overridden in some odd edge cases. Maybe better in the job setup/character spawn code?

Reply:
don't disagree. also really, really dont have a clue here though

Reply:
`spawnId` looks like a good proc? That's the code that handles renaming PDAs so makes some sense.